### ISSUE TITLE
Audit airplane usage + determine deprecation path

### ISSUE THREAD

**Issue by jnaviask** [hicommonwealth/commonwealth#5819]
# Description

Historically we have used airplane.dev for automating certain tasks for the growth team. Now that we have a proper internal admin panel, we should be able to migrate all configured airplane scripts over to that panel.

We should go into airplane and figure out which scripts, if any, need to be migrated, and plan out the work involved. Then we can deprecate airplane. The scripts involved can be found on the airplane site, with their details stored in a separate repo called `cw-internal-tools`, that we should aim to deprecate as well.

**Comment by kurtisassad** [comment]
Unused, can remove with no consequences:
custom-domain-heroku: We do this manually since it is a few button clicks. Should be set up in the admin panel, if we want to offload this minor task to growth.
discourse-import: Would break if ran because we don't have app cw-discourse-import on heroku.
heroku-acm-refresh: part of the custom domain procedure.
lead-gen-promotion: Discussed with mark, growth doesn't use. Also airplane usage statistics shown it not being run.
lead-gen-generation: Same as above

We only have one cron job that is run once a week:
email-digest-cron - We should move this onto the heroku timer like we do with Nakul's count calculation. The only cost is the compute it takes to run the server. This is an easy 1 pointer addition into heroku (Could fix real quickly if we decide to go for it)

**Comment by jnaviask** [comment]
@kurtisassad we do actually have a cw-discourse-import project on heroku -- does that change your evaluation of that script? Otherwise, sounds good, please ticket the email-digest-cron item as a ticket.

**Comment by kurtisassad** [comment]
@jnaviask Good catch. I did not see that. Basically this script is not run automatically (juding by the airplane usage logs). So we have two options depending on if we still use this discourse import utility:
If we don't plan to use in the future
- Delete the script and shut down the cw-discourse-import heroku app
If we do plan to use in the future
- Keep the heroku app, and move the script to commonwealth/scripts. Then just run one-off dynos in heroku manually when we want to perform this import.

**Comment by jnaviask** [comment]
We would like to be able to do discourse imports in the future, although we will need to clean + update the import script to do it. Can you make another ticket for course of action required to get it off airplane? Thanks!

**Comment by kurtisassad** [comment]
https://github.com/hicommonwealth/commonwealth/issues/5988
